packages/wallets/package.json

```diff
@@ -9,6 +9,8 @@
         "react": "^17.0.2"
     },
     "devDependencies": {
+        "@tanstack/eslint-plugin-query": "^4.34.1",
```

Comment:
@adrienne-deriv I believe we don't need this, Firstly `ReactQuery` is only used in `@deriv/api` directly, And secondly, everything is abstracted as generics behind `useFetch` and `useRequest` so we never gonna get any error from this ESLint rule since we people won't be using `useQuery` or `useMutation` at all 👀